packages/paste-website/src/pages/blog/2022-09-23-paste-newsletter.mdx

```diff
@@ -0,0 +1,456 @@
+      </Td>
+      <Td>
+        🟢 Green: Migrated our Visual Regression Testing suite from Applitools to Percy, improving test times by over
+        20% while additionally testing in each browser.
+        <br />
+        <br />
+        Expanded on our DangerJS PR checks to verify dependencies are pinned to prevent inconsistencies in builds.
+      </Td>
+    </Tr>
+    <Tr>
+      <Td>
+        <strong>Evolve the Twilio design language to support the vision of “One Twilio”</strong>
+      </Td>
+      <Td>
+        🟢 Green: After syncing with stakeholders and brand, along with reviewing the research findings, we narrowed
+        down our visual design concepts to a single direction and have been eagerly working on creating revamped
+        components in the new design language.
+        <br />
+        <br />
+        Next, we&apos;ll be using these revamped components to do some product stress testing that will help us
+        <ul>
+          <li>refine the language and correct inconsistencies</li>
+          <li>identify and document edge cases</li>
+          <li>cultivate an expert understanding of how the new design language should be used and implemented</li>
+        </ul>
+      </Td>
+    </Tr>
+    <Tr>
+      <Td>
+        <strong>Lay the groundwork for our new program management, content, and visual design disciplines</strong>
+      </Td>
+      <Td>De-scoped due to changing priorities.</Td>
+    </Tr>
+    <Tr>
+      <Td>
+        <strong>Build a UI kit for the Twilio Conversations API</strong>
+      </Td>
+      <Td>
+        🟢 Green: The Conversations UI Kit components are now included in the Paste Components library in Figma! We have
+        also released circular button sizes and the minimizable dialog component in code.
+      </Td>
+    </Tr>
+    <Tr>
+      <Td>
+        <strong>Paste goes international</strong>
+      </Td>
+      <Td>
+        ✅ Done: Paste now fully supports Twilio’s internationalization efforts for Q1. Paste components are capable of
+        supporting translated content and do not rely on any hard-coded strings.
+      </Td>
+    </Tr>
+  </TBody>
+</Table>
+
+<Box marginBottom="space120" />
+
+For more information, check out [our full BPM document](https://docs.google.com/document/d/1ok5cPj0JyysD5ukBXMbsyIVIdipw1tARzFiGTwGL-W4/edit?usp=sharing).
+
+## 👀 What's next
+
+<Table>
+  <THead>
+    <Tr>
+      <Th>Feature</Th>
+      <Th>Description</Th>
+    </Tr>
+  </THead>
+  <TBody>
+    <Tr>
+      <Td>
+        <strong>Multi-select Combobox and Combobox refined</strong>
+      </Td>
+      <Td>
+        Improvements to the existing <Anchor href="/components/combobox">combobox</Anchor> component, with a design
+        refresh and added functionality to select multiple items.
+      </Td>
+    </Tr>
+    <Tr>
+      <Td>
+        <strong>Avatar component, Square variant</strong>
+      </Td>
+      <Td>A new variant for the Avatar Component that creates a square shape.</Td>
```

Comment:
question: do you mind if we update this copy to saying `entity` variant? I took a v quick stab at the copy but feel free to change it up however

```suggestion
        <strong>Avatar component, entity variant</strong>
      </Td>
      <Td>A new variant for the Avatar Component for entities, like a business.</Td>
```